src/components/adminBar/TheAposAdminBarUser.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <AposContextMenu
+    class="apos-admin-user" tip-alignment="right"
+    :button="button" :menu="menu"
+  >
+    <template #prebutton>
+      <AposAvatar
+        class="apos-admin-user__avatar"
+        :src="avatarUrl"
+        size="32px" alt=""
+      />
+    </template>
+  </AposContextMenu>
+</template>
+
+<script>
+import AposContextMenu from '../contextMenu/AposContextMenu.vue';
+import AposHelpers from '../../mixins/AposHelpersMixin';
+import AposAvatar from '../../components/avatar/AposAvatar';
+
+export default {
+  components: {
+    AposContextMenu,
+    AposAvatar
+  },
+  mixins: [AposHelpers],
+  props: {
+    user: {
+      type: Object,
+      required: true
+    }
+  },
+  data() {
+    return {
+      avatarUrl: '',
+      menu: [
+        {
+          label: 'My Profile',
+          name: 'profile',
+          action: 'user-profile'
+        },
+        {
+          label: 'Log out',
+          name: 'logOut',
+          actions: 'user-logout'
+        }
+      ]
+    };
+  },
+  computed: {
+    button() {
+      return {
+        label: this.user.label || '',
+        icon: 'chevron-down-icon',
+        modifiers: ['icon-right'],
+        type: 'quiet'
+      };
+    }
+  },
+  mounted() {
+    this.avatarUrl = require('./userData').userAvatar;
```

Comment:
this should all be passed in from the story, the component should not rely on a file